perception/lidar_centerpoint/lib/postprocess/postprocess_kernel.cu

```diff
@@ -124,7 +124,7 @@ cudaError_t PostProcessCUDA::generateDetectedBoxes3D_launch(
     out_heatmap, out_offset, out_z, out_dim, out_rot, out_vel, config_.voxel_size_x_,
     config_.voxel_size_y_, config_.range_min_x_, config_.range_min_y_, config_.down_grid_size_x_,
     config_.down_grid_size_y_, config_.downsample_factor_, config_.class_size_,
-    config_.yaw_norm_threshold_, thrust::raw_pointer_cast(boxes3d_d_.data()));
+    config_.yaw_norm_thresholds_.data(), thrust::raw_pointer_cast(boxes3d_d_.data()));
```

Comment:
I get an illegal access error.
```
[lidar_centerpoint_node-22] terminate called after throwing an instance of 'thrust::system::system_error'
[lidar_centerpoint_node-22]   what():  reduce failed to synchronize: cudaErrorIllegalAddress: an illegal memory access was encountered
```
I think the vector needa to be copied to the device memory to access in the CUDA kernel function.
Here is an example: <https://docs.nvidia.com/cuda/thrust/index.html#iterators-and-static-dispatching>

Reply:
:bow:

Reply:
[b730489](https://github.com/autowarefoundation/autoware.universe/pull/1962/commits/b7304898f5a80b2bd381dd774dd67fddbca736de)